.travis.yml

```
@@ -4,24 +4,33 @@ language: haskell
 git:
   depth: 5
 
-cabal: "2.4"
+cabal: "3.0"
 
 cache:
   directories:
   - "$HOME/.cabal/store"
-  
+
 
 matrix:
   include:
-  - ghc: 8.6.5
-  
+  - ghc: 8.4.4
+    env: GHCVER='8.4.4' CABALVER='head'
+    os: linux
+    addons:
+      apt:
+        sources:
+        - hvr-ghcjs
+        packages:
+        - ghcjs-8.4.3
```

Let's try this GHC version. Maybe this one will work...
```suggestion
        - ghcjs-8.4
```